recipe/windows-data.patch

```diff
@@ -28,18 +28,3 @@ index 36f6587..83470e9 100644
  
  if(EXTRA_WARN)
    set(CMAKE_C_FLAGS "-Wall ${CMAKE_C_FLAGS}")
-diff --git a/poppler/GlobalParams.cc b/poppler/GlobalParams.cc
-index 5d1ef22..86e89bb 100644
---- a/poppler/GlobalParams.cc
-+++ b/poppler/GlobalParams.cc
-@@ -140,6 +140,10 @@ static const char *get_poppler_datadir(void)
```

Comment:
Are you intentionally removing this chunk of the `windows-data` patch?

Reply:
It seemed to me that this was already in the source but perhaps I've misread the diff? 
https://github.com/freedesktop/poppler/blob/poppler-21.09.0/poppler/GlobalParams.cc#L138-L142

Reply:
Unless I've confused myself, what the patch is doing is making it so that the code strips off an additional parent directory to get the correct location for the data files. This is a special need of conda-forge, as per the message in the original commit:

https://github.com/conda-forge/poppler-feedstock/commit/246b77ff762e3af0cb78c7d882ca3d19a4afd38f